Sources/XCMetricsBackendLib/Config/routes.swift

```diff
@@ -43,7 +43,7 @@ func routes(_ app: Application) throws {
     app.middleware.use(cors)
 
     // Log File Repository
-    let logFileRepository = LogFileRepositoryFactory.makeWithConfiguration(config: config, logger: app.logger)
+    let logFileRepository = LogFileRepositoryFactory.makeWithConfiguration(config: config, logger: app.logger, client: app.client)
```

Comment:
I need access to Vapor's client object so I can send HTTP requests to Azure using the client configuration. The other log repositories (S3 and GCP) don't make requests directly, but Azure doesn't have a Swift library available.

Reply:
Why not instantiate the client via the constructor logic to avoid modifying the signature, as opposed to passing the client directly?

Reply:
The client instance that I'm injecting is already configured somewhere else. This configuration is intended to apply throughout the application and, by creating another instance within the constructor, we'd have to configure it again.  I think this would be a bit redundant. The same applies to the logger.

Additionally, the `makeWithConfiguration` function is not called anywhere else, so altering its signature shouldn't be an issue in this case.